Makefile

```
@@ -8,6 +8,8 @@ ifeq ($(TARGETOS),windows)
 	BIN_SUFFIX := .exe
 endif
 
+DESTDIR ?= dist
```

```suggestion
DIST_DIR ?= dist
```

We should remove the file from `cspell`s ignore list